InventoryManagementSystem/Product.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InventoryManagementSystem
+{
+    internal class Product
+    {
+        public string Name { get; set; }
+        public int Price { get; set; }
+        public int Quantity { get; set; }
+        public Product(String name, int price, int quantity)
```

Comment:
usually we don't need to write a constructor for such classes that contains only data. we use the C# object initializer feature, 
check: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers